src/commands/initProjectForVSCode/pythonSteps/addPythonInitVSCodeSteps.ts

```diff
@@ -34,6 +38,9 @@ export async function addPythonInitVSCodeSteps(
         } else {
             promptSteps.push(new PythonVenvListStep(venvs));
         }
+    } else if (createPythonVenv) {
```

Comment:
Looks like there's a few test failures. You can ignore the tests titled "Template Count - Staging Python", but the other two are relevant to this PR.

- Create New Python Project skip venv

Looks like this one actually found a bug. If you do "Create New Project" and select "skip venv", you'll get the same prompt twice. Since "init for vs code" always happens for "create new project", I think you can remove these steps from [here](https://github.com/microsoft/vscode-azurefunctions/blob/master/src/commands/createNewProject/pythonSteps/addPythonCreateProjectSteps.ts#L23)

- Create New Python Project skip venv

Just need to update the test to reflect the behavior change. Similar to [this test](https://github.com/microsoft/vscode-azurefunctions/blob/master/test/project/createNewPythonProject.test.ts#L16), you can specify `inputs: [/skip/i]` [here](https://github.com/microsoft/vscode-azurefunctions/blob/master/test/project/initProjectForVSCode.test.ts#L57)

Reply:
Thanks for the clarifications about the tests. Was not sure how to interpret the result when master were failing as well. Will try to have a look at the bug this weekend or next week.

Reply:
I believe my latest push fixed the open issues.